src/hooks/useScrolled.ts

```diff
@@ -5,7 +5,7 @@ export function useScrolled(elementId: string) {
   const [isScrolled, setIsScrolled] = useState(true);
 
   useEffect(() => {
-    let DELAY_STICKY_TRANSITION = window.innerWidth < 600 ? 700 : 300;
+    let DELAY_STICKY_TRANSITION = window.innerWidth < 767 ? 0 : 300;
```

Comment:
Is it possible to use the values in `breakpoints.ts`? I know those are string values with the added "px" but just for consistency. This is not a blocker for this PR though.

Reply:
This inspired me to just make those breakpoint values numbers.. I *think* that makes more sense overall